library/src/main/java/com/heartlandpaymentsystems/library/terminals/c2x/C2XDevice.java

```diff
@@ -233,6 +255,34 @@ public void cancelTransaction() {
         transactionManager.cancel();
     }
 
+    //OTA methods
+    public void getAvailableTerminalVersions(TerminalUpdateType terminalUpdateType) {
```

Comment:
@philip-white-heartland We should be using an enum from a `com.heartlandpaymentsystems` package to keep branding consistent. Also, their naming is confusing since `TerminalUpdateType.KERNEL` represents config updates (according to [this](https://github.globalpay.com/VitalPOS/tsysemv/blob/1ae98cff6b7c12d2f827e7a2896af79ec7b98af2/Android/hardware/bbpos/src/main/java/com/tsys/payments/hardware/bbpos/BbposController.java#L894-L901)), so we will want to work around that as well.